__tests__/api-paths.test.js

```diff
@@ -108,3 +97,51 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+describe("GET /api/reviews/:review_id", () => {
+  test("status: 200 returns an array with review objects", () => {
+    return request(app)
+      .get("/api/reviews/3")
+      .expect(200)
+      .then((data) => {
+        const reviews = data.body.reviews;
+        reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              title: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_id: 3,
+              review_img_url: expect.any(String),
+              category: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              review_body: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  test("should return only the required review", () => {
+    return request(app)
+      .get("/api/reviews/4")
+      .expect(200)
+      .then((data) => {
+        const result = data.body.reviews;
+        expect(result.length).toBe(1);
+      });
+  });
+});
+
+describe("error handeling", () => {
+  test("status:404 when serching for an incorret path", () => {
+    return request(app)
+      .get("/api/not_a_path")
+      .expect(404)
+      .then((data) => {
+        expect(data.body.msg).toBe("Path not found");
+      });
+  });
+  test("status:400 when serching for a missing id in reviews", () => {
```

Comment:
Word to the wise for grouping tests - I would personally recommend grouping both the `happy` and `sad` paths for each endpoint together in one describe block - it seems logical to have all `error handling` tests together but as you get to the point of having 40+ tests it starts to get hard to find the ones for the endpoint you're currently working on

Reply:
ive move the relavent id test